src/modules/table.marko

```diff
@@ -0,0 +1,945 @@
+                            <td class="table__cell table__cell--numeric">305</td>
+                            <td class="table__cell table__cell--numeric">$27.99</td>
+                            <td class="table__cell">FREE</td>
+                            <td class="table__cell">4/9 - 4/11</td>
+                        </tr>
+                        <tr>
+                            <td class="table__cell">
+                                <span class="checkbox">
+                                    <input
+                                        aria-label="Select row"
+                                        class="checkbox__control"
+                                        type="checkbox"
+                                        name="checkbox-default"
+                                        >
+                                    <span class="checkbox__icon" hidden>
+                                        <svg
+                                            class="checkbox__unchecked"
+                                            height="18"
+                                            width="18"
+                                            aria-hidden="true"
+                                        >
+                                            <icon-symbol name="checkbox-unchecked-18"/>
+                                        </svg>
+                                        <svg
+                                            class="checkbox__checked"
+                                            height="18"
+                                            width="18"
+                                            aria-hidden="true"
+                                        >
+                                            <icon-symbol name="checkbox-checked-18"/>
+                                        </svg>
+                                    </span>
+                                </span>
+                            </td>
+                            <th class="table__cell" scope="row">Sony</th>
+                            <td class="table__cell">Playstation 5</td>
+                            <td class="table__cell"></td>
+                            <td class="table__cell table__cell--numeric">$519.99</td>
+                            <td class="table__cell table__cell--numeric">205</td>
+                            <td class="table__cell table__cell--numeric">265</td>
+                            <td class="table__cell table__cell--numeric">199</td>
+                            <td class="table__cell table__cell--numeric">$29.99</td>
+                            <td class="table__cell">FREE</td>
+                            <td class="table__cell">4/11 - 4/15</td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+<div class="table table--mode-selection" tabindex="0">
```

Comment:
The existence of this tab index means the example code is in the tab sequence. Probably not what we intended.